public/app/features/dashboard/utils/getPanelChromeProps.tsx

```diff
@@ -79,7 +79,7 @@ export function getPanelChromeProps(props: CommonProps) {
 
   const onCancelQuery = () => {
     props.panel.getQueryRunner().cancelQuery();
-    DashboardInteractions.panelCancelQueryClicked();
```

Comment:
why do we need this?

Reply:
I am assuming we want it, but we lost it in the refactor like the others, but tbh, I don't know.

Reply:
I would just keep it as it is right now. We're not sure

Reply:
Ok, maybe @dprokop can help us here. 
Was [this change](https://github.com/grafana/grafana/pull/79078/files#diff-2e4d7a90c2dcc990dd68e1cc1e882ff084d676c94bc5f70f4a78424a2bd25bb9L81) intended or is a regression?
I found that now we are calling this event from scenes [here](https://github.com/grafana/grafana/blob/main/public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts#L511) so we should update both if we decide to restore it.

Reply:
@evictorero - it's a regression, sorry!

Reply:
This is ok to be modified then. @evictorero Just approved